Vagrantfile

```diff
@@ -81,7 +92,7 @@ Vagrant.configure("2") do |config|
 	config.vm.define "dev" do |dev|
 
 		dev.vm.box = "ubuntu/bionic64"
-		dev.vm.network "forwarded_port", guest: 80, host: 5000
+		dev.vm.network "forwarded_port", guest: ENV['APP_GUEST_PORT'], host: ENV['application_host_port']
 
```

Comment:
here the variable in lowercase

Reply:
and please shorten the variable name. APPLICATION=>APP

Reply:
done